packages/swapper/src/swappers/ZrxSwapper/getZrxTradeQuote/getZrxTradeQuote.ts

```diff
@@ -170,7 +170,7 @@ async function _getZrxTradePseudoQuote(
   try {
     return Ok({
       id: uuid(),
-      receiveAddress,
+      receiveAddress: undefined,
```

Comment:
This was a blatant mistake - receiveAddress is the discriminator and should be undefined for rates

Reply:
The file name is now off, I took 1 min trying to understand why you were talking about rates in this comment while being in the `getZrxTradeQuote` file

Reply:
Yeah I had the same thought, want might want to put it into a separate `getZrxTradeRate` or something file.

Reply:
gm 3622358529